### PR TITLE
github: Disable TICS temporarily

### PR DESCRIPTION
**Pull request by tomponline**
As per https://discourse.canonical.com/t/end-of-the-year-update-for-tiobe-tics-required-action-to-disable-tics-cicd-pipelines/4890

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -308,7 +308,8 @@ jobs:
       CGO_LDFLAGS: "-L/home/runner/go/bin/dqlite/libs/"
       LD_LIBRARY_PATH: "/home/runner/go/bin/dqlite/libs/"
       CGO_LDFLAGS_ALLOW: "(-Wl,-wrap,pthread_create)|(-Wl,-z,now)"
-    if: ${{ ( github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' ) && github.ref_name == 'main' && github.repository == 'canonical/lxd' }}
+    #if: ${{ ( github.event_name == 'workflow_dispatch' || github.event_name == 'schedule' ) && github.ref_name == 'main' && github.repository == 'canonical/lxd' }}
```
Comment (roosterfish) [inline .github/workflows/tests.yml:311]: If you just remove the `if`the job will always run no?

**Review by roosterfish — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (tomponline) [reply .github/workflows/tests.yml:311]: Yes but we want it to never run

Comment (tomponline) [reply .github/workflows/tests.yml:311]: Hence the if false

Comment (roosterfish) [reply .github/workflows/tests.yml:311]: Ah I was looking too fast and missed the second line. My bad, looks good.

**Review by roosterfish — APPROVED** [review]

**Pull request merged**